devsite/devsite/seed.py

```diff
@@ -152,7 +155,7 @@ def enroll_date(max_days_back):
 
     for user in users:
         if VERBOSE:
-            print('seeding course enrollment for user {}'.format(user.username))
+            print(('seeding course enrollment for user {}'.format(user.username)))
```

Comment:
Minor: Why we're adding extra parenthesis around the formatted string?
```suggestion
            print('seeding course enrollment for user {}'.format(user.username))
```